packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -124,6 +125,15 @@ const getBillingStepStatus = createSelector(
             };
         }
 
+        const isGooglePayBillingAddressEditingEnabled = isExperimentEnabled(
+            config?.checkoutSettings,
+            'experiment name will be added here after adding it in LD',
+        );
+        const isUsingGooglePay =
+            (checkout && checkout.payments
+                ? checkout.payments.some((payment) => (payment?.providerId || '').startsWith('googlepay'))
+                : false) && isGooglePayBillingAddressEditingEnabled;
```

Comment:
Would be better to check `isGooglePayBillingAddressEditingEnabled` first in this statement

`const isUsingGooglePay = isGooglePayBillingAddressEditingEnabled && ...`

Reply:
Done